src/Hack.sol

```diff
@@ -583,8 +603,8 @@ contract GuessTheAverage {
     function distribute(uint256 _count) public {
         require(currentStage == Stage.WinnersFound, "Winners must have been found");
         for (uint256 i = cursorDistribute; i < winners.length && (_count == 0 || i < cursorDistribute + _count); i++) {
-            // Send ether to the winners, use send not to block.
-            payable(winners[i]).send(totalBalance / (winners.length - numberOfLosers));
+            // Send ether to the winners, use send not to block, if one of the account cannot receive ETH.
```

Comment:
should we write it?

Reply:
I think it's ok to keep it, so that they don't focus on that for too long